openhtf/core/measurements.py

```diff
@@ -348,6 +348,12 @@ def __init__(self, description='', unit=units.NO_DIMENSION):
     self.description = description
     self.unit = unit
 
+  def __eq__(self, other):
+    return (self.description == other.description and self.unit == other.unit)
+
+  def __str__(self):
+    return str(self._asdict())
```

Comment:
Let's use the pattern commonly used in OpenHTF of including the class name.

Also I think this should be \_\_repr__ not \_\_str__. And \_\_str__ will then default to \_\_repr__.

```python
def __repr__(self):
  return '<%s: %s>' % (type(self).__name__, self._asdict())
```

Reply:
(meant to “request changes)

Reply:
Done.